# Fix header ImathMatrix.h when used directly by other software

diff -N -r -U 3 ilmbase-1.0.2-orig/Imath/ImathMatrix.h ilmbase-1.0.2/Imath/ImathMatrix.h
--- ilmbase-1.0.2-orig/Imath/ImathMatrix.h	2010-07-17 01:48:40.000000000 +0300
+++ ilmbase-1.0.2/Imath/ImathMatrix.h	2010-08-28 12:33:56.000000000 +0300
@@ -49,6 +49,7 @@
 #include "ImathVec.h"
 #include "ImathShear.h"
 
+#include <cstring>
 #include <iostream>
 #include <iomanip>
 
